api/v1alpha1/profilesubscription_types.go

```diff
@@ -36,17 +36,17 @@ type ProfileSubscriptionSpec struct {
 	// +optional
 	Branch string `json:"branch,omitempty"`
 
-	// Path is the location in the git repo containing the profile definition. Only used in combination with Branch
+	// Path is the location in the git repo containing the profile definition
 	// +optional
 	Path string `json:"path,omitempty"`
 
 	// Values holds the values for the Helm chart specified in the first artifact
 	// +optional
 	Values *apiextensionsv1.JSON `json:"values,omitempty"`
 
-	// Version is the git tag containing the profile definition
+	// Tag is the git tag containing the profile definition
 	// +optional
-	Version string `json:"version,omitempty"`
+	Tag string `json:"tag,omitempty"`
 
```

Comment:
since this information is just for us, having the `tag` and not the `version` is more useful. `version` is repeated below in the catalog reference